src/importexport/capella/internal/capella.cpp

```diff
@@ -463,14 +463,14 @@ static bool findChordRests(BasicDrawObj const* const o, Score* score, const int
         if (seg->segmentType() != SegmentType::ChordRest) {
             continue;
         }
-        ChordRest* cr = static_cast<ChordRest*>(seg->element(track));
+        ChordRest* cr = toChordRest(seg->element(track));
         if (cr) {
-            if (graceNumber1 > 0) {       // the spanner is starting from a grace note
-                Chord* chord = static_cast<Chord*>(cr);
-                foreach (Chord* cc, chord->graceNotes()) {
+            if ((graceNumber1 > 0) && cr->isChord()) {       // the spanner is starting from a grace note
```

Comment:
This additional check is the core of this fix, the rest is more or less cosmetic.
(And that change is basically 'stolen' from further down)